content/en/docs/tasks/run-application/run-single-instance-stateful-application.md

```diff
@@ -133,7 +133,7 @@ behind a Service and you don't intend to increase the number of Pods.
 Run a MySQL client to connect to the server:
 
 ```
-kubectl run -it --rm --image=mysql:5.6 --restart=Never mysql-client -- mysql -h mysql -ppassword
+kubectl run -it --rm --image=mysql:5.6 --restart=Never mysql-client -- mysql -h mysql -p password
```

Comment:
[If I remember correctly], `mysql` expects no space between the `-p` and the password that follows it.